Makefile

```diff
@@ -140,8 +140,8 @@ pytest-saas: compose-build
 .PHONY: black
 black: compose-build
 	@echo "Running black formatting against the src/ directory..."
-	@docker-compose run $(IMAGE_NAME) \
-	black src/
+	@docker-compose run $(IMAGE_NAME) black src/
```

Comment:
Would it be possible to run just `black .` instead since the projects  `pyproject.toml` has a list of files and directories to exclude? It might be slightly faster than spinning up the project twice to run `black`

Reply:
The nuance here is that we also only want to run it on `src/` and `tests/` to avoid formatting things like virtualenvs.